spec.yml

```diff
@@ -2029,6 +2038,97 @@ paths:
           schema:
             $ref: '#/definitions/License'
 
+  /teams/:
+    x-resource: Teams
+    get:
+      summary: Get a list of teams
+      tags:
+        - Users
+      parameters:
+        - $ref: '#/parameters/pageSize'
+        - $ref: '#/parameters/page'
+      responses:
+        200:
+          description: SUCCESS
+          schema:
+            $ref: '#/definitions/TeamsPaginated'
+    post:
+      summary: Create a team
+      tags:
+        - Users
+      parameters:
+        - name: team
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/Team'
+      responses:
+        201:
+          description: Get a newly created team
+          schema:
+            $ref: '#/definitions/Team'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+  /teams/{uuid}:
+    x-resource: Teams
+    get:
+      summary: Get team details
+      tags:
+        - User
+      parameters:
+        - $ref: '#/parameters/uuid'
+      responses:
+        200:
+          description: Detail about a team
+          schema:
+            $ref: '#/definitions/Team'
+        404:
+          description: |
+            UUID parameter does not refer to a team or the user is not able to view the team it refers to
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      summary: Update a team
+      tags:
+        - User
+      parameters:
+        - name: team
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/Team'
```

Comment:
We may also want to use a schema that better reflects the fields that the user can actually update. I'm sure we have other spots in this spec where we don't do this, but it's something that could be important if we want to move to spec based testing of the API.